crates/oxc_minifier/src/ast_passes/peephole_fold_constants.rs

```diff
@@ -17,6 +6,16 @@ use crate::{
     ty::Ty,
     CompressorPass,
 };
+use num_bigint::BigInt;
+use oxc_ast::ast::*;
+use oxc_span::{GetSpan, Span, SPAN};
+use oxc_syntax::number::ToJsInt32;
+use oxc_syntax::{
+    number::NumberBase,
+    operator::{BinaryOperator, LogicalOperator, UnaryOperator},
+};
+use oxc_traverse::{Ancestor, Traverse, TraverseCtx};
+use std::cmp::Ordering;
```

Comment:
These import statements are grouped and ordered from furthest to closes, did your IDE reorder these?

Reply:
Yes, it is :joy:

Reply:
Please revert this irrelevant change :-)

Reply:
Got it.